app/assets/scss/partials/_app.scss

```diff
@@ -2,6 +2,10 @@
   background-color: $govuk-body-background-colour;
 }
 
+.govuk-width-container--wide {
```

Comment:
If we do go down this route the application specific class should use the namespace `app-` since this class is not in core GOV.UK Frontend.

https://design-system.service.gov.uk/get-started/extending-and-modifying-components/